frontend/public/components/overview/pods-overview.tsx

```diff
@@ -0,0 +1,149 @@
+import * as _ from 'lodash-es';
+import * as React from 'react';
+import { Link } from 'react-router-dom';
+import { ListGroup } from 'patternfly-react';
+
+import { ResourceLink, resourcePath, SidebarSectionHeading, StatusIconAndText } from '../utils';
+import { podPhase, PodKind, K8sResourceKind } from '../../module/k8s';
+
+const kind: string = 'Pod';
+const MAX_PODS: number = 3;
+
+const podUpdateTime = (pod: PodKind ) => {
+  const allStatuses = [
+    ..._.get(pod, 'status.containerStatuses', []),
+    ..._.get(pod, 'status.initContainerStatuses', []),
+  ];
+  const updateTimes = _.reduce(allStatuses, (times, nextStatus) => {
+    if (nextStatus.state.running) {
+      return [...times, _.get(nextStatus, 'state.running.startedAt')];
+    }
+    if (nextStatus.state.terminated) {
+      return [...times, _.get(nextStatus, 'state.terminated.finishedAt')];
+    }
+    if (nextStatus.lastState.running) {
+      return [...times, _.get(nextStatus, 'lastState.running.startedAt')];
+    }
+    if (nextStatus.lastState.terminated) {
+      return [...times, _.get(nextStatus, 'lastState.terminated.finishedAt')];
+    }
+    return [...times, _.get(nextStatus, _.get(pod, 'startTime'))];
+  }, []);
+
+  return _.head(_.reverse(updateTimes.sort()));
+};
+
+const errorPhases = [
```

Comment:
We should have one consistent list.

https://github.com/openshift/console/blob/master/frontend/public/components/overview/index.tsx#L46-L47

Reply:
that seems like a real short list. I went with the phases that cause an error icon for status, determined here: https://github.com/openshift/console/blob/master/frontend/public/components/utils/status-icon.tsx#L30

I definitely agree there should be one place because in topology it shows differently as well (the donut chart has instances that use warnings while the pod status shows an error icon)

Reply:
Could this also be bug'd and done as a follow on?